tests/smoke-test.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/bash
+set -e
+
+GODWOKEN__KICKER_PATH=$(cat test-config.json | jq '."godwoken-kicker-path"' | tr -d '"')
+GODWOKEN__ETH_ADDRESS=$(cat test-config.json | jq '."godwoken-eth-address"' | tr -d '"')
+GODWOKEN__PRIVATE_KEY=$(cat test-config.json | jq '."godwoken-private-key"' | tr -d '"')
+
+set +e
+GODWOKEN__IS_CHAIN_UP=$(curl -s localhost:6101 | jq .status)
+set -e
+
+if [ -z "$GODWOKEN__IS_CHAIN_UP" ]; then
+  pushd $GODWOKEN__KICKER_PATH
+
+  sudo make init
```

Comment:
Why we need sudo?
Correctly configured docker doesnt need sudo.
Come visit me tomorrow we will configure it for you, or google how to configure it. Here lets not use sudo